{{cookiecutter.__project_slug}}/{{cookiecutter.module_name}}/main.py

```diff
@@ -0,0 +1,16 @@
+from {{cookiecutter.module_name}}.processing import filter_outliers
+import pandas as pd
+
+INPUT_FILE_PATH = './example_input.csv'
+OUTPUT_PATH = './output.csv'
+THRESHOLD = 5
+COLUMN_NAME = 'column_1'
+
+try:
+    data = pd.read_csv(INPUT_FILE_PATH)
+except FileNotFoundError:
+    print(f"File '{INPUT_FILE_PATH}' not found. Make sure the file exists.")
+    exit()
+
```

Comment:
consider using try...except....else here instead if calling exit()

Reply:
Great tip, changed it.